test/PrintAsObjC/error.swift

```diff
@@ -1,18 +1,23 @@
 // RUN: %empty-directory(%t)
 // RUN: %target-swift-frontend(mock-sdk: %clang-importer-sdk) -emit-module-path %t/error.swiftmodule -emit-objc-header-path %t/error.h -experimental-allow-module-with-compiler-errors %s
-// RUN: %FileCheck %s < %t/error.h
+// RUN: %FileCheck --input-file %t/error.h %s
 // RUN: %check-in-clang %t/error.h
 
 // REQUIRES: objc_interop
 
 import Foundation
 
+// TODO: Ideally we'd output invalid decls regardless (so that they could eg. be used in code
+// completion), but we avoid doing so for now to prevent crashes. Revisit later to handle a possibly
+// invalid AST while printing the ObjectiveC header.
```

Comment:
Is there a bug report for this task? If not, there probably should be. If so, you should probably cite the bug number.

Reply:
There was internally, I've added SR-15088 and linked it now. Thanks!